pkg/autoscaler/bucket/bucket.go

```diff
@@ -29,3 +33,19 @@ func IsBucketHost(host string) bool {
 	// have host with the prefix. Maybe use regexp for improvement.
 	return strings.HasPrefix(host, prefix)
 }
+
+// AutoscaleBucketName returns the name of the Autoscaler bucket with given `ordinal`
+// and `total` bucket count.
+func AutoscaleBucketName(ordinal, total uint32) string {
+	return strings.ToLower(fmt.Sprintf("%s-%02d-of-%02d", prefix, ordinal, total))
```

Comment:
just out of interest, any specific reason to pad the numbers here rather than just `%s-%d-of-%d`, or just looks nicer?

Reply:
To be consistent with https://github.com/knative/pkg/blob/master/leaderelection/context.go#L191